Contributor list

ultrasaurus opened the pull request:
This list acknowledges people who contributed to the draft spec before it was on github,
and also allows us to include people who make substantive contributions in other forums,
such as the Serverless WG meetings where participation is tracked outside of github.

Also, it is very helpful to have easy reference to the projects that contributors are working on
where they use (or hope to use) CloudEvents.  

Closes #42

ultrasaurus commented:
oops -- meant to make this separate from the starter doc -- I can refactor as an independent
commit if this proposal is accepted.

Fixes https://github.com/cloudevents/spec/issues/42

ultrasaurus commented:
fixed this so it only has the contributor list file (and also fixed up formatting a bit)

duglin commented on about/contributors.md:
```diff
@@ -0,0 +1,25 @@
+  * Arun Gupta, Ajay Nair, Rob Leidle, Orr Weinstein
+* **Google**
+  * [Google Cloud Functions](https://cloud.google.com/functions/) and [Cloud Functions for Firebase](https://firebase.google.com/docs/functions/)
+  * [Sarah Allen](https://github.com/ultrasaurus), [Rachel Myers](https://github.com/rachelmyers) (Serverless WG)
```
I don't see the pattern here w.r.t. when a new bullet is used. Can we either put all people from one company on the same bullet or give each person their own bullet?

leecalcote replied:
Products are listed at the same level that people are. Not all have a product. Should they be listed? Are products contributors?

duglin replied:
I'd prefer if we didn't list them.  And to be honest, I'm not sure I want to even have a "people list". If people want to see who contributed then they can look at the git logs.  No different than a git code repo.

ac360 replied:
In my opinion, our big challenge is to gain traction with this effort, otherwise it will fade away and our wonderful effort will be for nothing.  Fortunately, we have some huge assets to help us on our journey: *the CNCF* and *the quality of people who are working on this already*.  

The fact that so many incredible people (and orgs) are collaborating on this sends a clear signal to the market that something of significance is happening.  Making these names hard to access is akin to removing the marquee customers on the landing page of your product or hiding the celebrity names on a movie poster.  I believe we will be handicapping ourselves in a major way by not making these people clearly visible.  Further, maintaining a Contributor list that is clearly visible shows that we welcome, recognize and celebrate our contributors, and it will be helpful in seeking new contributors.  We should therefore make these names as easy to see as possible and a Contributor list is a great way to do this.

Being explicit about these methods always feels awkward.  But building products/traction/community is a science.  It will take great product, but also great packaging.

ac360 replied:
That said, we need to make sure these people are ok with being listed.  Including a clear statement on this page that says: **“Contributions do not constitute an official endorsement”** will greatly help here.

Also, listing products feels like it will add a higher burden of maintenance and complexity.  Already, there are some products and team members listed here that are out of date.  Maybe you can optionally allow for people to put their name and the product they work on?  Also, perhaps their twitter handle so that others can contact them?

**Example:**

* **Serverless Inc.**
   * Austen Collins - Serverless Framework - @austencollins

duglin replied:
What's the bar for being added? Company, product or people-wise ?

ac360 replied:
@duglin Perhaps it's as simple as they contribute to the Github repository.

duglin replied:
maybe. Just feels like we're setting the board up for a game  :-) 

I'm not keen on listing products - I don't see the point of that. Products are not contributing, companies and people are.

ultrasaurus replied:
The products provide context for why the companies are contributing, and actually more relevant than the companies themselves.

notque replied:
If the bar is contributing, I should be listed. Not arguing either way, but it's an omission if that is the bar.

ultrasaurus replied:
@notque picking this back up... I will add you in my updated PR!

duglin commented on about/contributors.md:
```diff
@@ -0,0 +1,25 @@
+  * Austen Collins, Rupak Ganguly (Serverless WG)
+  * Brian Neisler, Jeremy Coffield, Ganesh Radhakirshnan
+* **IBM**
+  * [Doug Davis](https://github.com/duglin), [Daniel Krook](https://github.com/krook), Matt Rutkowski (Serverless WG)
```
I guess if we're listing products please list IBM Cloud Functions: https://console.bluemix.net/openwhisk/

duglin commented:
there are outstanding comments/questions

ultrasaurus commented:
I think I've added everyone who provided their info in the comments.  I didn't go through the github contributors list, since I added text that invites everyone to add their own info.

Please review!

leecalcote commented:
Looking good! I'm not sure if @carimura has a link to include for Fn?

duglin commented on about/contributors.md:
```diff
@@ -0,0 +1,48 @@
+  * [IBM Cloud Functions](https://console.bluemix.net/openwhisk/)
+  * Doug Davis, [@duglin](https://github.com/duglin)
+  * Daniel Krook, [@krook](https://github.com/krook)
+  * Matt Rutkowski, Michael M Behrendt
```
please use mrutkows for matt
and michael-behrendt for Michael.

duglin commented on about/contributors.md:
```diff
@@ -0,0 +1,48 @@
+  * Orit Nissan-Messing
+* **Microsoft**
+  * [Microsoft Event Grid](https://azure.microsoft.com/en-us/services/event-grid/)
+  * Bahram Banisadr, Clemens Vasters, Cesar Ruiz-Meraz, Dan Rosanova, Raja Ravipati
```
banisadr for Bahram
clemensv for Clemens
djrosanova for Dan

duglin commented on about/contributors.md:
```diff
@@ -0,0 +1,48 @@
+  * [Microsoft Event Grid](https://azure.microsoft.com/en-us/services/event-grid/)
+  * Bahram Banisadr, Clemens Vasters, Cesar Ruiz-Meraz, Dan Rosanova, Raja Ravipati
+* **Oracle**
+  * Travis Reeder, Chad Arimura
```
carimura for Chad
treeder for Travis

carimura replied:
**Oracle**
* [Fn Project](https://fnproject.io)
* Chad Arimura [@carimura](https://github.com/carimura)
* Stanley Halka [@shalka](https://github.com/shalka)
* Travis Reeder [@treeder](https://github.com/treeder)

duglin commented on about/contributors.md:
```diff
@@ -0,0 +1,48 @@
+* **Serverless Inc**
+  * [Serverless Framework and Event Gateway](https://serverless.com/)
+  * Austen Collins - [@ac360](https://github.com/ac360)
+  * Rupak Ganguly, Brian Neisler, Jeremy Coffield, Ganesh Radhakirshnan
```
rupakg for Rupak
brianneisler for Brian

markpeek commented:
How about adding:

**VMware**
[Dispatch Functions Framework](https://vmware.github.io/dispatch)
Mark Peek - [@markpeek](https://github.com/markpeek)

cathyhongzhang commented:
@ultrasaurus LGTM. Thanks!

cathyhongzhang commented:
@ultrasaurus , Sorry just found out that I forgot to add the other Huawei Sererless product I am working on---"Function Graph". Here is the link https://www.huaweicloud.com/en-us/product/functiongraph.html. Could you add this together with Huawei Function Stage https://www.huaweicloud.com/en-us/product/functionstage.html? Thanks!

duglin commented:
@ultrasaurus if you update this one for @cathyhongzhang I think its fair game for tomorrow and should be an easy one.

ac360 commented:
LGTM

duglin commented:
Approved on 3/29 call.
